### PR TITLE
Refactor/nodekill

### PR DESCRIPTION
**Pull request by 8000meters**
refactored dtrace routers to run NodeKill when the user starts the app and dtrace/perf at the same time. Nodekill will now run immediately after dtrace/perf ends.

### CODE REVIEW

File: src/controllers/application.controller.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { spawn, execSync } from 'child_process';
 import { existsSync } from 'fs-extra';
 import { InfernodeError } from '../utils/globalErrorHandler';
 import logger from '../utils/logging';
+import { resourceLimits } from 'worker_threads';
```
Comment (seadubs) [inline src/controllers/application.controller.ts:8]: Looks like there is an unused import that can be removed.

**Review by seadubs — CHANGES_REQUESTED** [review]

File: src/controllers/application.controller.ts (hunk 1 above)
Comment (8000meters) [reply src/controllers/application.controller.ts:8]: I have no idea what that is or where it came from. is that not something you added?

Comment (seadubs) [reply src/controllers/application.controller.ts:8]: I don't think so. Maybe a type resulting in an auto import at some point? VSCode can be pretty aggressive about that sometimes.

Comment (8000meters) [reply src/controllers/application.controller.ts:8]: weird. I'll get rid of it.

**Review by seadubs — APPROVED** [review]

**Pull request merged**